CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## [Unreleased]
+
+## [0.1.0] - 2023-01-30
```

Comment:
Update changelog to real date when taking out of draft

Reply:
I reset the changelog, so naturally will have to update it in a release prep PR post merge as expected.